posts/Guillaume/2017-03-30-hibernate-validator-600-alpha2-out.adoc

```diff
@@ -0,0 +1,131 @@
+= "Hibernate Validator 6.0.0.Alpha2 is out with improved Bean Validation 2.0 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Hibernate Validator 6 is going to be the Reference Implementation of Bean Validation 2.0. This Alpha2 release is coordinated with the Early Draft 2 release of the Bean Validation 2.0 specification.
+
+It is also a playground used to validate future enhancements of the Bean Validation specification so feedback on the subjects presented here is very welcome!
+
+Note that Hibernate Validator 6 requires JDK 8 or above.
+
+If you missed the Alpha1 release, consulting http://in.relation.to/2017/02/16/hibernate-validator-600-alpha1-out/[its announcement] might be beneficial to you before continuing your reading.
+
+== What's new since Alpha1
+
+=== Improved value extraction support
```

Comment:
Let's rather speak of validation of (generic) containers than value extraction. Or "container element validation". I.e. thinking of it from the user's perspective, what they gain, rather than how it's implemented.